src/nextlabs_sdk/_cloudaz/_engine/_constructors.py

```
@@ -22,18 +22,19 @@ def _build_query_plan(
+    query_params: dict[str, int | str] = {dialect.page_param: page_no}
     if page_size is not None:
         query_params[dialect.size_param] = page_size
     return RequestPlan("GET", path, params=query_params, json=None)
 
 
 def _build_search_plan(
-    path: str,
+    path_template: str,
     args: Mapping[str, object],
     page_no: int,
     page_size: int | None,  # noqa: WPS110
```

Ruff (0.15.21)

[warning] Invalid rule code in `# noqa`: WPS110

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)